example/App.tsx

```diff
@@ -1,54 +1,73 @@
-import * as React from 'react';
-import { StyleSheet, View, Text, Image, Animated, Button } from 'react-native';
+import React, { useRef, useState } from 'react';
+import { View, Text, Image, Animated, Button } from 'react-native';
 // @ts-ignore
 import { ReactNativeZoomableView } from '@openspacelabs/react-native-zoomable-view';
+import { styles } from './style';
 
 export default function App() {
-  const zoomAnimatedValue = React.useRef(new Animated.Value(1)).current;
+  const zoomAnimatedValue = useRef(new Animated.Value(1)).current;
   const scale = Animated.divide(1, zoomAnimatedValue);
-  const [showMarkers, setShowMarkers] = React.useState(true);
+  const [showMarkers, setShowMarkers] = useState(true);
+  const [size, setSize] = useState<{ width: number; height: number }>();
+
+  // Use layout event to get centre point, to set the pin
+  const [pin, setPin] = useState({ x: 0, y: 0 });
+  // Debounce the change event to avoid layout event firing too often while dragging
+  const debouncedUpdatePin = debounce(setPin, 10);
+
+  const staticPinPosition = size
+    ? { left: size.width / 2, top: size.height / 2 }
+    : undefined;
 
   return (
     <View style={styles.container}>
       <Text>ReactNativeZoomableView</Text>
-      <View style={styles.box}>
+      <View style={styles.box} onLayout={(e) => setSize(e.nativeEvent.layout)}>
         <ReactNativeZoomableView
+          // Where to put the pin in the content view
+          staticPinPosition={staticPinPosition}
+          // Callback that returns the position of the pin
+          // on the actual source image
+          disableMomentum
+          onStaticPinPositionChange={debouncedUpdatePin}
+          onStaticPinPositionMove={debouncedUpdatePin}
           maxZoom={30}
           initialZoom={1.5}
           // Give these to the zoomable view so it can apply the boundaries around the actual content.
           // Need to make sure the content is actually centered and the width and height are
           // measured when it's rendered naturally. Not the intrinsic sizes.
-          // For example, an image with an intrinsic size of 400x200 will be rendered as 300x150 in this case.
-          // Therefore, we'll feed the zoomable view the 300x100 size.
-          contentWidth={300}
-          contentHeight={150}
-          panBoundaryPadding={50}
+          contentWidth={400}
+          contentHeight={400}
```

Comment:
might be good to use the `applyContainResizeMode` here

Reply:
yup!